build-telemetry-ffi.sh

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env bash
```

Comment:
I was diffing this and `build-profiling-ffi.sh` and noticed this is missing the changes from https://github.com/DataDog/libdatadog/pull/101 . Not a blocker for this PR, but you may want to import them as well :)